### PR TITLE
Danish translations provided

### PR DESCRIPTION
**Pull request by thkir**
- [ ] This change has been discussed with the author.
- [ ] It implements / fixes issue #.

---

**Description:**

I followed to guide to submit the translations. 

Hope it's okay!

Best regards

Thomas

### CODE REVIEW

File: iina/da.lproj/QuickSettingViewController.strings [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+"uqb-mz-A22.title" = "BAGGRUND";
+
+/* Class = "NSTextFieldCell"; title = "Table View Cell"; ObjectID = "vDh-JM-5Wz"; */
+"vDh-JM-5Wz.title" = "Table View Cell";
+
+/* Class = "NSButtonCell"; title = "Custom..."; ObjectID = "vFD-HU-RVz"; */
+"vFD-HU-RVz.title" = "Brugerdefineret...";
+
+/* Class = "NSTextFieldCell"; title = "Sub style options may break ASS rendering, and some of them will be disabled depending on subtitle type."; ObjectID = "wBD-pZ-Bvu"; */
+"wBD-pZ-Bvu.title" = "Sub style options may break ASS rendering, and some of them will be disabled depending on subtitle type.";
```
Comment (lhc70000) [inline iina/da.lproj/QuickSettingViewController.strings]: Not translated

File: iina/da.lproj/QuickSettingViewController.strings [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+
+/* Class = "NSTextFieldCell"; title = "0s"; ObjectID = "wgA-op-JP4"; */
+"wgA-op-JP4.title" = "0s";
+
+/* Class = "NSTextFieldCell"; title = "Audio track:"; ObjectID = "x39-62-7KC"; */
+"x39-62-7KC.title" = "Audio track:";
```
Comment (lhc70000) [inline iina/da.lproj/QuickSettingViewController.strings]: Not translated

File: iina/da.lproj/KeyBinding.strings [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+"opt.volume" = "Volumen";
+"opt.mute" = "Mute";
+"opt.audio-delay" = "Lyd forsinkelse";
+
+"opt.sub" = "Sub track";
+"opt.sub-delay" = "Sub delay";
+"opt.sub-pos" = "Sub pos";
+"opt.sub-scale" = "Sub scale";
+"opt.sub-visibility" = "Sub visibility";
+"opt.sub-ass-style-override" = "ASS style override";
```
Comment (lhc70000) [inline iina/da.lproj/KeyBinding.strings]: I'm not sure but these seems untranslated

**Comment by thkir** [comment]
Hi again

Seems a bit strange...

I will just revise it again and update!

Sorry!

Best regards

Thomas

**Comment by thkir** [comment]
Sorry for my lack of knowledge here. Should I close this request and add a new one?

I have found mistakes in 5 of the 31 files! They are now corrected and ready! 

Best regards

Thomas

**Comment by lhc70000** [comment]
Thanks Thomas! Now there should be no problem and I'm going to merge it. Since the next version will be released soon, It would be great if you could also update the translations for newly added features then.

**Comment by thkir** [comment]
Sure i will! How it that working, do i get a list of need translations or how do you usually manage that.?

**Pull request merged**